vcbuild.bat

```diff
@@ -147,6 +149,7 @@ if defined build_release (
   set licensertf=1
   set download_arg="--download=all"
   set i18n_arg=small-icu
+  set cctest=1
```

Comment:
AFAIK on the release CI (where `build_release` is used) tests aren't run. Anyone from @nodejs/releasers / @nodejs/build / @nodejs/platform-windows able to confirm?

Reply:
They are probably not, but I guess someone somewhere might depend on that. This change will not make big change in terms of building release binaries, it is mostly aimed to help out when developing Node.